### PR TITLE
Styling change pages

### PR DESCRIPTION
**Pull request by rslaughter12**
Added consistent styling to the reset password and user profile pages

### CODE REVIEW

File: client/src/pages/UserProfile.js [hunk 1]
```diff
@@ -141,21 +147,18 @@ export function UserProfile() {
...
-          }
+          onChange={(e) => setEmail(e.target.value.trim())}
         />
-        <Button
-          type='submit'
-          variant='contained'
-        >
-          Save
+        <Button type='submit' variant='contained'>
+          Save New Username
```
Comment (sendusyourbones) [inline client/src/pages/UserProfile.js]: They can also update their email address. Maybe just "save changes"?

File: client/src/pages/UserProfile.js [hunk 2]
```diff
@@ -108,31 +118,27 @@ export function UserProfile() {
...
+        sx={{ mt: 3, '& > *': { mb: 2} }}
       >
-        <Box>
-          <Typography component='h2' variant='h6'>
-            Password
-          </Typography>
-          <ResetPassword textInput={'Reset Password'} email={email} />
-        </Box>
+        <Box></Box>
         <TextField
```
Comment (sendusyourbones) [inline client/src/pages/UserProfile.js]: Can we add some space between the two input fields (I like how the change password page looks!)?

File: client/src/pages/UserProfile.js (hunk 1 above)
Comment (sendusyourbones) [reply client/src/pages/UserProfile.js]: Also, can we add some space between the buttons and the field above, as well as between each of the buttons?

File: client/src/components/DeleteButton.js [hunk 3]
```diff
@@ -98,7 +98,7 @@ export function DeleteButton(props) {
         startIcon={<DeleteIcon />}
         color='secondary'
       >
-        Delete
+        Delete Profile
```
Comment (sendusyourbones) [inline client/src/components/DeleteButton.js]: This component is also used to delete a lesson, so we need some conditional stuff or a variable here in order for it to show properly in both places.

File: client/src/pages/UserProfile.js (hunk 2 above)
Comment (rslaughter12) [reply client/src/pages/UserProfile.js]: input fields are updated

File: client/src/components/DeleteButton.js (hunk 3 above)
Comment (rslaughter12) [reply client/src/components/DeleteButton.js]: we will just keep it as delete no biggie, switched it back

File: client/src/pages/UserProfile.js (hunk 1 above)
Comment (rslaughter12) [reply client/src/pages/UserProfile.js]: the button styling is not gonna happen lol, they are struggling to communicate for some reason.

**Pull request closed without merging**